_posts/en/newsletters/2020-08-05-newsletter.md

```diff
@@ -59,7 +59,32 @@ FIXME:harding to update Tuesday
 [Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
 (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
 
-- [Bitcoin Core #19569][] Enable fetching of orphan parents from wtxid peers FIXME:moneyball
+- [Bitcoin Core #19569][] allows Bitcoin Core to fetch the parents of _orphan_
+  transactions from peers that relay transactions using wtxid. An orphan
+  transaction is an unconfirmed transaction that we receive from a peer where we
+  don't yet have the parent transactions, either as part of our best block
+  chain, or in the mempool. More precisely, an orphan transaction has at least
+  one transaction input whose associated output is not in the UTXO set or our
+  mempool's outpoint map.
+
+    When we receive an orphan transaction, we place it in a temporary data
+    structure called the orphan set. We then ask the peer that sent us the
+    orphan to also send us the parent transactions that we don't yet have. We can
+    do that because the orphan refers to the txids of the parent transactions. We
```

Comment:
Suggest s/refers/contains/ and /the parent/its parent.  Also the same suggested s/refer/contain/ change in the next paragraph.  Although protocol experts know that transactions are relayed without any extra metadata, many readers might not know that and so think we're talking about relay-related metadata.  This might make them wonder why wtxid relay doesn't simply replace the parent txid references with wtxid references.  I think making it clear that the txid reference is an internal part of the transaction (which hopefully readers know can't be changed), it'll be clearer why we can't simply swap one for the other.

Reply:
Good suggestion!